pombola/core/tests/test_views.py

```diff
@@ -293,110 +293,3 @@ def test_letters_link_to_the_right_url_if_query_param_present(self):
         self.assertEqual(link, '?a=1&order=name&letter=P')
         self.assertTrue(response.context['alphabetical_link_from_query_parameter'])
 
-
-class TestPersonView(WebTest):
```

Comment:
Removed these tests because we have specific tests for the South African profile page. [`SAPersonProfileSubPageTest`](https://github.com/OpenUpSA/pombola/blob/master/pombola/south_africa/tests.py#L1257)